toolchain/check/eval.cpp

```diff
@@ -617,7 +617,9 @@ static auto GetConstantFacetTypeInfo(EvalContext& eval_context,
                                      SemIR::LocId loc_id,
                                      const SemIR::FacetTypeInfo& orig,
                                      Phase* phase) -> SemIR::FacetTypeInfo {
-  SemIR::FacetTypeInfo info;
+  // TODO: Process other requirements.
```

Comment:
This TODO looks a little lost/unclear now, but maybe we can keep the `other_requirements` around (see comments below). Otherwise, it should be made more clear what it's trying to say.